mlir/test/Dialect/Arith/eliminate-explicit-rounding.mlir

```diff
@@ -0,0 +1,92 @@
+// RUN: mlir-opt %s --split-input-file -math-legalize-to-f32 --arith-emulate-unsupported-floats="source-types=bf16 target-type=f32" -eliminate-explicit-rounding | FileCheck %s
```

Comment:
When testing, it is recommended to only call your pass. This way, the results of your pass will be more clearly visible.

Reply:
Thanks for your suggestion. Fixed.